### DISCUSSION TITLE
what is the objective function in pFBA using forward+backward fluxes?

### DISCUSSION THREAD

**Discussion by shaicoh3n** [opencobra/cobrapy#1376] [General]
I know that the objective function is to minimize the sum of the absolute of the fluxes. I'm wondering if that means using the forward (v_f) and backward (v_b) fluxes it will be sum over |v_f-v_b| or |v_f+v_b| or something else?

**Comment by cdiener** [comment]
It's the sum over v_f+v_b. That works for minimization (only) because there `min | v_f - v_b | = k` always implies a solution where either v_f or v_b must be zero and the other one will be `+-k`.

**Reply by shaicoh3n** [reply]
thx, It is v_b+v_f :)

**Comment by shaicoh3n** [comment]
It is v_b+v_f :)
